exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,34 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
-
+function isNegative(num) {
+  if (num < 0) {
+    return true;
+  } else {
+    return false;
```

Comment:
Thank you Ekrem.

With this specific function, we only need to check if a number is negative or positive, I think Ekrem you were looking at a different function 👍 

What you have done Shima works perfectly! We could simplify this by just returning the comparison, if you recall from lessons, something like `(num < 0)` will return true or false. Knowing this we can do something like this:

```javascript
function isNegative(num) {
  return (num < 0);
}
```

Since `num < 0` will be true or false we get the result we want :)

Reply:
Thank you, Stephen
You are right 🙈 I should have written this for exercises/F-predicates/exercise.js.
To review for our buddy group friends' PR was Home Work last week :)

Reply:
You did a fantastic job and I am grateful for you helping out :) Submitting PR reviews is a big part of being a developer, you smashed it mate 👍 😄